lib/reference/keywords.mjs

```diff
@@ -103,11 +103,11 @@ export const fontWeightKeywords = uniteSets(
 
 const fontStyleKeywords = uniteSets(basicKeywords, ['normal', 'italic', 'oblique']);
 
+/** @todo add sub, super, text, emoji, unicode */
```

Comment:
This will be handled in a future issue/PR.
i.e. this is a refactor PR but I don't want this to be forgotten